packages/suite-desktop-core/webpack/core.rspack.config.ts

```diff
@@ -82,24 +82,25 @@ const config: webpack.Configuration = {
         ...dependencies,
         ...devDependencies,
         'bufferutil', // optional dependency of ws lib
-        'memcpy', // optional depencency of bytebuffer lib
         'utf-8-validate', // optional dependency of ws lib
-        'osx-temperature-sensor', // optional dependency of systeminformation lib
     ],
+    externalsType: 'commonjs2',
     module: {
         rules: [
             {
-                test: /\.js$/,
-                exclude: /node_modules/,
-                use: ['babel-loader'],
-            },
-            {
-                test: /\.ts$/,
-                exclude: /node_modules/,
-                use: {
-                    loader: 'babel-loader',
-                    options: {
-                        presets: ['@babel/preset-typescript'],
+                test: /\.(js|ts)$/,
+                exclude: [/[\\/]node_modules[\\/]/],
+                loader: 'builtin:swc-loader',
+                options: {
+                    jsc: {
+                        parser: {
+                            syntax: 'typescript',
+                        },
+                        externalHelpers: true,
+                    },
+                    env: {
+                        // electron 32 uses node 20 https://www.electronjs.org/docs/latest/tutorial/electron-timelines
+                        targets: 'node 20',
```

Comment:
Every time we update electron, because the target runtime is dependent on nodeJS version that electron uses.
But it does not change very often..
This is crucial though, you can target an older version of node, it will just use older syntax which may not be optimal performance-wise but idk how important that is :shrug: